optimization/imagify-reset-optimization-status/imagify-reset-optimization-status.php

```diff
@@ -1,30 +1,33 @@
 <?php
 /**
- * Plugin Name: Imagify | Reset Optimization Status
- * Description: Will “reset” Imagify’s optimization status in the database, so that previously optimized images will be considered not optimized. Physical image files will not actually be modified! How to use: 1. Activate plugin. 2. Reload plugin page once. 3. Deactivate plugin!
- * Plugin URI:  https://github.com/wp-media/imagify-helpers/tree/master/optimization/imagify-reset-optimization-status/
- * Author:      Imagify Support Team
- * Author URI:  http://imagify.io/
- * License:     GNU General Public License v2 or later
- * License URI: http://www.gnu.org/licenses/gpl-2.0.html
+ * Plugin Name:  Imagify | Reset Optimization Status
+ * Description:  Will “reset” Imagify’s optimization status in the database, so that previously optimized images will be considered not optimized. Physical image files will not actually be modified! How to use: 1. Activate plugin. 2. Reload plugin page once. 3. Deactivate plugin!
+ * Plugin URI:   https://github.com/wp-media/imagify-helpers/tree/master/optimization/imagify-reset-optimization-status/
+ * Version:      1.0.1
+ * Requires PHP: 5.3
+ * Author:       Imagify Support Team
+ * Author URI:   http://imagify.io/
+ * License:      GPLv2
+ * License URI:  http://www.gnu.org/licenses/gpl-2.0.html
  *
- * Copyright SAS WP MEDIA 2018
+ * Copyright 2020 WP Media
+ *
+ * @package WP-Media\ImagifyPluginHelpers\ResetOptimizationStatus
  */
-namespace ImagifyPlugin\Helpers\optimization\status;
 
-defined( 'ABSPATH' ) or die();
+namespace ImagifyPluginHelpers\Optimization\ResetOptimizationStatus;
```

Comment:
The namespace and `@package` should match.

```php
WPMedia\ImagifyPluginHelpers\Optimization\ResetOptimizationStatus
```

Reply:
Can we use `WP-Media` in the package name (it is already used everywhere) and `WPMedia` in the namespace?

Reply:
This should give the following:
```php
 * @package WP-Media\ImagifyPluginHelpers\OptimizationResetOptimizationStatus
...
namespace WPMedia\ImagifyPluginHelpers\Optimization\ResetOptimizationStatus;
```

Reply:
@Screenfeed we are moving to a PSR-4 namespace strategy where there's no `-` or `_`. 

And then with `@package` it should match the namespace, if we are including `@package`. That said, `@package` is redundant when we use a namespace. The namespace itself is the package identifier.